tests/python3-libraries/python3-libraries.yaml

```diff
@@ -0,0 +1,14 @@
+schemaVersion: "1.0.0"
+commandTests:
+  - name: "env var setup"
+    envVars:
+      - key: "VIRTUAL_ENV"
+        value: "/env"
+      - key: "PATH"
+        value: "/env/bin:$PATH"
+    command: ["echo", "'setup complete'"]
+
+  - name: "requirements"
+    setup: [["virtualenv", "-p", "/opt/python3.5/bin/python3.5", "/env"]]
```

Comment:
I added python3.5 to the $PATH, so change "/opt/python3.5/bin/python3.5" to "python3.5" here.